tests/_support/Module/CleanUpModule.php

```diff
@@ -25,6 +25,24 @@ class CleanUpModule extends Module
      * @return void
      */
     public function _before(TestInterface $test): void
+    {
+        $this->removeCreatedFiles();
+    }
+
+    /**
+     * @param \Codeception\TestInterface $test
+     *
+     * @return void
+     */
+    public function _after(TestInterface $test): void
```

Comment:
When the test fails you can not look into the generated file anymore as it is directly removed. That's the reason why you should always clean up before the tests.